jobs/transforms/user_audits.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beam functions and transforms to provide validation for user models."""
```

Comment:
In what way is ValidateUserModelId (the only thing in this class) a "transform"?

Reply:
Ah, `DoFn` counts as a function so that's why it's here. There will be "transforms" added later, presumably, so I want to keep the doc string consistent even though there aren't any yet.

Reply:
Hm, I don't see any functions here.

On further investigation, it looks like beam.DoFn is actually a class. Could you change the docstring to `Beam functions (subclasses of beam.DoFn) and transforms to ...`?

Reply:
Done.